src/api/ServerMetadata.ts

```diff
@@ -100,11 +100,21 @@ export class ServerMetadata {
     }
   }
 
-  /** Returs a convenient data structure with all capabilities listed. */
+  /** Does this version support the drift 2.0 flows enhancements? */
+  public enhancedFlows() {
+    if (this.type && this.type === ServerTypes.MERIDIAN) {
+      return this.version.ge('2020.0.0');
```

Comment:
I made an assumption on the version here,

Reply:
We're not releasing 2019 until the newer Drift stuff is in, AFAIK.  @j-white  ?

Reply:
Yep, it should be 2019

Reply:
Ok, fixed it to 2019.